### ISSUE TITLE
double submit of /wsapi/stage_user on main site signup

### ISSUE THREAD

**Issue by jrgm** [mozilla/persona#1828]
I was signing up a new user on the main site page for login.dev.anosrep.org. I received email with a link to /verify_email_address and token of
https://login.dev.anosrep.org/verify_email_address?token=AW3wEnj3MVqMknrX3ppuIw8upq8S3Iar3QZUn7NZCUIj1m4a

Trying to use that link in the same browser that I started with, I get this in the web page:

```
"""There was a problem with your signup link.
   Has this address already been registered?
   Please close this window and try again."""
```

and on the network, I can see a GET for
https://login.dev.anosrep.org/wsapi/email_for_token?token=AW3wEnj3MVqMknrX3ppuIw8upq8S3Iar3QZUn7NZCUIj1m4a
which returns 200 OK with a payload of '{"success":false,"reason":"no such secret"}'

In the database on login.dev.anosrep.org in the staged table, there is an entry from the email address I was using, but the token is indeed not the one I got in email.

But then duh, I see that I had two emails in the inbox of that user, both sent out at the same time, with one arriving a couple of seconds after the other. So I expect that this is probably a side-effect of GH-1813.

**Comment by shane-tomlinson** [comment]
Same cause as issue #1813.  Fixed with PR #1814.  Closing.

**Comment by jrgm** [comment]
verified same issue as GH-1813 and fixed.
